### DISCUSSION TITLE
artalk-no-comment-text/

### DISCUSSION THREAD

**Discussion by ghost** [eallion/eallion.com#108] [Giscus]
# artalk-no-comment-text/

这篇文章介绍了作者如何在Artalk的管理后台中实现了一个功能，即在无评论时随机显示诗词。作者在前端配置中插入了今日诗词的API代码，并选择了适合不同背景颜色的文字颜色。作者表示这种低代码实现方式无需侵入网页的JS，也无需单独编写兼容CSS，这个功能在作者参与的多个网页或应用中都嵌入了今日诗词的API。

https://www.eallion.com/artalk-no-comment-text/

**Comment by fishhh666** [comment]
🧐哇这个好有意思！wp.com 免费套餐正好是没法使用 css js 这种的，这个正好避开了限制。看了下文档还有定制推荐，好天才的想法

**Reply by eallion** [reply]
有用就好！😊
